code-challenges.test.js

```diff
@@ -10,42 +10,225 @@
+  
+
+
+// Attempt #940523 (the closest "dynamic" solution without being the solution)
+// This solution prints the full sentence with only the first letter of the first name capitalized.
+// aboutPeople = (array) =>{
+  //   return array.map(
+  //     value => `${value.name[0].toUpperCase()}` +
+  //             `${value.name.substring(1)}` + 
+  //             ` is ${value.occupation}.` )
+  // }
+
+
+
+  aboutPeople = (array) => {
+    return array.map(
+      ({name, occupation}) => {
+        twoNames = name.split(" ")
+        return (
+          twoNames[0][0].toUpperCase() + 
+          twoNames[0].substring(1) + " " +
+          twoNames[1][0].toUpperCase() + 
+          twoNames[1].substring(1) + " is " +
+          occupation + "."
+        ) 
+      }
+    )
+  }
+
+// ******* GREEN FAILURE *******
+// PASS  ./code-challenges.test.js
+// aboutPeople
+//   ✓ takes in an array of objects and returns an array with a sentence about each person with their name capitalized (1 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+  
+
+// Research that helped provide clarity for the solution above: https://stackoverflow.com/questions/58509981/destructuring-an-array-of-objects-es6
+
+
+
+
+
+
+
+// --------------------2) 
+// Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
+
+
 
 // a) Create a test with an expect statement using the variables provided.
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+
+describe("onlyRemainders", (array) => {
+  it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+    const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+    const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+      expect(onlyRemainders(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+      expect(onlyRemainders(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+
+// ******* RED FAILURE *******
+// ● onlyNums › takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3
+// ReferenceError: onlyNums is not defined
+
+
 
 
 // b) Create the function that makes the test pass.
 
 
-// --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
+// ******* PSUEDOCODE *******
+// input = array with mixed data types 
+  // filter array for integer data type only
+  // modulo 3 for each integer
+// output = array with only integer data type  
+  // array contains the remainders of numbers from the original array divided by 3
+  // return the results of each integer modulo 3
+
+
+  onlyRemainders = (array) => {
+    let onlyNums = array.filter(value => typeof value === "number")
+    return onlyNums.map(value => value % 3)
+  }
```

Comment:
This looks great!  One thing you may want to keep in mind is to start your functions with the keyword const.